python-main.js

```diff
@@ -1125,12 +1159,16 @@ function web_editor(config) {
         vex.open({
             content: Mustache.render(template, context),
             afterOpen: function(vexContent) {
-                $(vexContent).find('.snippet-selection').click(function(e){
-                    var snippet_name = $(this).find('.snippet-name').text();
-                    EDITOR.triggerSnippet(snippet_name);
-                    vex.close();
-                    EDITOR.focus();
-                });
+                focusModal();
+                $(vexContent).find('.snippet-selection').on('click keydown', (function(e){
+                    if(e.type == 'click' || e.type=='keydown' && e.which == 32 || e.which == 13) {
```

Comment:
I would add extra parenthesis for clarity:

```suggestion
                    if(e.type == 'click' || (e.type=='keydown' && (e.which == 32 || e.which == 13))) {
```

Is the order above correct?

Reply:
I've added this suggestion and also put the keys in numeric order